components/phoenix_ev_charger.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Phoenix EV Charger",
+  "owner": ["@abrlox"],
+  "manifest": "https://raw.githubusercontent.com/abrlox/home-assistant-phoenix-ev-charger/main/custom_components/phoenix_ev_charger/manifest.json",
```

Comment:
Your `pymodbus` requirement is conflicting with the one used in HA Core
<https://github.com/home-assistant/core/blob/dev/requirements_all.txt#L1557>
Please use `pymodbus>=2.5.1`

Reply:
Hi ludeeus,

Thanks for the help.
I changed the requirement as requested. Sorry I didn't notice earlier.